### PR TITLE
feat: ratings env vars

### PR DESCRIPTION
**Pull request by matthew-hagemann**
Closes #1386

### CODE REVIEW

File: snap-store_0+git.6adffcd-dirty_amd64.snap
Comment (tim-hm) [inline snap-store_0+git.6adffcd-dirty_amd64.snap:1]: Did this artefact get included by accident?

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -46,6 +46,9 @@ parts:
 apps:
   snap-store:
     command: bin/snap-store
+    environment:
+      RATINGS_SERVICE_URL: 'ratings.ubuntu.com'
+      RATINGS_SERVICE_PORT: '8080'
```
Comment (tim-hm) [inline snap/snapcraft.yaml:51]: Won't the api be at `https://ratings.ubuntu.com` (ie port 443 although this is usually implied by the protocol)

**Review by tim-hm — CHANGES_REQUESTED** [review]

File: snap-store_0+git.6adffcd-dirty_amd64.snap
Comment (matthew-hagemann) [reply snap-store_0+git.6adffcd-dirty_amd64.snap:1]: Oh, yes it did, ty!

File: test/config_service_test.dart [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    configService.load();
+    expect(configService.ratingServiceUrl, 'localhost');
+    expect(configService.ratingsServicePort, 8080);
+  });
+
+  test('env vars set', () {
+    final configService = ConfigService();
+    configService.testEnvironment = {
+      'RATING_SERVICE_URL': 'test.url',
+      'RATING_SERVICE_PORT': '1234'
```
[the remark this answers is not in the document]
Comment (matthew-hagemann) [reply test/config_service_test.dart:17]: Ah, thank you :smile:

**Review by d-loose — APPROVED** [review]

**Review by tim-hm — APPROVED** [review]

**Pull request merged**